fluentd/fluent-plugin-grafana-loki/lib/fluent/plugin/out_loki.rb

```diff
@@ -99,6 +113,21 @@ def write(chunk)
         opts = {
           use_ssl: uri.scheme == 'https'
         }
+
+        if !@cert.nil? && !@key.nil?
+          opts = opts.merge(
+            verify_mode: OpenSSL::SSL::VERIFY_PEER,
```

Comment:
Should we handle `VERIFY_NONE` case for self-signed sertificates?

Reply:
If we're using self-signed certificates, wouldn't it be better to pass `ca_cert` to verify the server's self-signed cert?

Reply:
Hmm, you are right.
Please leave it as-is until `VERIFY_NONE` use case is found.